pkg/steps/multi_stage.go

```diff
@@ -430,6 +430,12 @@ func (s *multiStageTestStep) generatePods(steps []api.LiteralTestStep, env []cor
 		pod.Labels[MultiStageTestLabel] = s.name
 		pod.Spec.ServiceAccountName = s.name
 		pod.Spec.TerminationGracePeriodSeconds = terminationGracePeriodSeconds
+		if step.DNSConfig != nil {
```

Comment:
Can you please add a test where this is exercised?

Reply:
Yes, i am working on the test.